subsys/bluetooth/common/Kconfig

```diff
@@ -96,7 +96,7 @@ config BT_BUF_ACL_RX_COUNT
 	int "Number of incoming ACL data buffers"
 	default NET_BUF_RX_COUNT if NET_L2_BT
 	default 3 if BT_RECV_BLOCKING
-	default 6
+	default 32
```

Comment:
this is a rather large increase for all applications. They probably don't use BT shell so they don't actually need that much.
Please set this in your app's `prj.conf` instead, and in `tests/bluetooth/shell/prj.conf`.